app/lib/features/home/pages/dashboard.dart

```diff
@@ -39,11 +39,11 @@ class Dashboard extends ConsumerWidget {
                         searchWidget(context),
                         featuresNav(context),
                         const SizedBox(height: 20),
+                        const MySpacesSection(limit: 5),
+                        const SizedBox(height: 20),
                         const MyTasksSection(limit: 5),
                         const SizedBox(height: 28),
                         const MyEventsSection(limit: 5),
-                        const SizedBox(height: 20),
-                        const MySpacesSection(limit: 5),
```

Comment:
why have those moved up? I think the others are more important if available than the myspaces ...

Reply:
Sorry I merged it without seeing your this comment.

I am thinking to move up spaces section till we implement bookmarked spaces in quick action page but feel free to put my space section in bottom in any of the upcoming PRs.

Reply:
I feel that my-events and my-tasks should trump the space list, because it means I have interacted with them and that there is now content with higher importance (and urgency).